API.pm

```diff
@@ -494,11 +494,13 @@ sub getFileInfo {
 
 	if ($format =~ /fl.c/i) {
 		$preferredFormat = $prefs->get('preferredFormat');
-		if ($preferredFormat < QOBUZ_STREAMING_FLAC_HIRES || ($maxSupportedSamplerate && $maxSupportedSamplerate <= 48_000)) {
+		if ($preferredFormat < QOBUZ_STREAMING_FLAC_HIRES || ($maxSupportedSamplerate && $maxSupportedSamplerate < 48_000)) {
 			$preferredFormat = QOBUZ_STREAMING_FLAC;
 		}
 		elsif ($preferredFormat > QOBUZ_STREAMING_FLAC_HIRES) {
-			$preferredFormat = QOBUZ_STREAMING_FLAC_HIRES2;
+        		if ($maxSupportedSamplerate && $maxSupportedSamplerate <= 96_000) {
```

Comment:
Shouldn't we use `QOBUZ_STREAMING_FLAC_HIRES2` at some point still?

Please use tabs, not spaces. Click the "Show whitespace" item at the head of the "Files changed" tab if you don't see spaces vs. tasbs. Thanks!

Reply:
OK, replaced the spaces (was only on one line, I think?) - I'm using the GitHub online editor for this one, hope that's not messing anything up.

`$preferredFormat` can only be `QOBUZ_STREAMING_FLAC_HIRES2` if it's > `QOBUZ_STREAMING_FLAC_HIRES`, so that line was always superfluous.

Reply:
Oh, right!